experimental/packages/otlp-proto-exporter-base/tsconfig.esnext.json

```diff
@@ -6,10 +6,7 @@
     "rootDir": "src",
     "tsBuildInfoFile": "build/esnext/tsconfig.esnext.tsbuildinfo"
   },
-  "include": [
-    "src/**/*.ts",
-    "src/generated/*.js"
-  ],
+  "include": ["src/**/*.ts", "src/generated/*.js"],
```

Comment:
Is this formatting change intended? Maybe auto generated?

Reply:
It is auto-generated.